runtime/shibuya/src/lib.rs

```diff
@@ -1522,7 +1526,13 @@ parameter_types! {
 impl pallet_migrations::Config for Runtime {
     type RuntimeEvent = RuntimeEvent;
     #[cfg(not(feature = "runtime-benchmarks"))]
-    type Migrations = ();
+    type Migrations = (
```

Comment:
Those migrations have already been executed on Shibuya, and will be deleted once you have rebased the master branch